README.md

```diff
@@ -34,7 +34,7 @@ DEBUG_HTTP=true
   - [ ] [Duplicated keys](https://github.com/mgrachev/dotenv-linter/issues/5);
   - [ ] [Lowercase keys](https://github.com/mgrachev/dotenv-linter/issues/6);
   - [ ] [Keys without values](https://github.com/mgrachev/dotenv-linter/issues/7);
-  - [ ] [Incorrect delimiter](https://github.com/mgrachev/dotenv-linter/issues/8);
+  - [x] Incorrect delimiter
```

Comment:
Please, move it up and add an example like this https://github.com/mgrachev/dotenv-linter/blob/master/README.md#leading-space